pkgs/development/libraries/duckdb/default.nix

```diff
@@ -37,6 +37,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DDUCKDB_EXTENSION_CONFIGS=${src}/.github/config/in_tree_extensions.cmake"
+    "-DENABLE_EXTENSION_AUTOINSTALL=1"
+    "-DENABLE_EXTENSION_AUTOLOADING=1"
```

Comment:
I'd agree with that.

Reply:
i think that either way you're going to get some sort of error with or without autoload. without it you'll be trying to use a missing feature, so that should generate an error. with it you'll error out trying to download the plugin.

however, we are still building all the available plugins for the library / cli and the default ones + httpfs for the python build.

since we do build all the in tree plugins for the library / cli that are [autoloadable](https://github.com/duckdb/duckdb/blob/401c8061c6ece35949cac58c7770cc755710ca86/scripts/generate_extensions_function.py#L298) the pluggins should never get downloaded anyway so these flags are kind of no-ops.

however -- for python build we are bundling just the default extensions + httpfs, so there are a few which could trigger an autoinstall / load which are:

- autocomplete
- excel
- inet (i think this one is ip address manipulation)
- sqlsmith
- visualizer


My personal preference is to have the flags off and not have things autoload / autoinstall so I'm happy to disable these again.